enterprise/frontend/src/embedding-sdk-bundle/components/public/SdkQuestion/SdkQuestion.unit.spec.tsx

```diff
@@ -325,8 +325,7 @@ describe("InteractiveQuestion", () => {
 
     const columnTitle = screen.getByTestId("column_title");
     await userEvent.clear(columnTitle);
-    await userEvent.type(columnTitle, "A New Test Column");
-    await userEvent.tab();
```

Comment:
One interesting thing is that, we don't need `tab` anymore. As we debounce the change.

https://github.com/metabase/metabase/blob/8638e0271824a49135412cd779f92f03f9dd552e/frontend/src/metabase/visualizations/components/settings/ChartSettingInput.tsx#L28-L31

however, moving this statement after the expect, which doesn't make sense by the way, reduces the duration from almost 7s to less than 100ms. I don't know why that's the case. But we don't need to tab to have the change reflected in the UI anymore, so I remove it.